### PR TITLE
French translation of setupWizardConfigureInstance

### PR DESCRIPTION
**Pull request by SoleneGK**
The setupWizardConfigureInstance_fr.properties file has been entirely translated in french.
wizard_fr

### CODE REVIEW

File: core/src/main/resources/jenkins/install/SetupWizard/setupWizardConfigureInstance_fr.properties [hunk 1]
```diff
@@ -1,2 +1,6 @@
 Instance\ Configuration=Configuration de l''instance
-Jenkins\ URL=Jenkins URL
+Jenkins\ URL=URL Jenkins 
+jenkinsURL_help=L''URL Jenkins est utilis\u00e9e pour fournir l''URL de base pour les liens absolus vers les diverses ressources Jenkins. \
```
Comment (aHenryJard) [inline core/src/main/resources/jenkins/install/SetupWizard/setupWizardConfigureInstance_fr.properties:3]:
```suggestion
jenkinsURL_help=L''URL de Jenkins est utilis\u00e9e pour fournir l''URL de base pour les liens absolus vers les diverses ressources Jenkins. \
```

**Review by aHenryJard — APPROVED** [review]

File: core/src/main/resources/jenkins/install/SetupWizard/setupWizardConfigureInstance_fr.properties [hunk 2]
```diff
@@ -1,2 +1,6 @@
 Instance\ Configuration=Configuration de l''instance
-Jenkins\ URL=Jenkins URL
+Jenkins\ URL=URL de Jenkins 
+jenkinsURL_help=L''URL de Jenkins est utilis\u00e9e pour fournir l''URL de base pour les liens absolus vers les diverses ressources Jenkins. \
```
Comment (PierreBtz) [inline core/src/main/resources/jenkins/install/SetupWizard/setupWizardConfigureInstance_fr.properties:3]: Il faudrait peut être soit traduire PR, soit au moins l'écrire en toutes lettres, j'ai déjà vu des gens qui ne connaissaient pas le terme.

Comment (SoleneGK) [reply core/src/main/resources/jenkins/install/SetupWizard/setupWizardConfigureInstance_fr.properties:3]: J'ai enlevé l'abbréviation pour mettre le terme complet.

**Review by Wadeck — APPROVED** [review]
As the original author of the root URL configuration properties, it covers what I wanted 👍

**Review by oleg-nenashev — APPROVED** [review]
LGTM, thanks!

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Comment by timja** [comment]
may as well get it in.

**Pull request merged**